plume-models/src/medias.rs

```diff
@@ -102,19 +102,19 @@ impl Media {
     pub fn html(&self, conn: &Connection) -> Result<SafeString> {
         let url = self.url(conn)?;
         Ok(match self.category() {
-            MediaCategory::Image => SafeString::new(&format!(
+            MediaCategory::Image => SafeString::trusted(&format!(
                 r#"<img src="{}" alt="{}" title="{}">"#,
                 url, escape(&self.alt_text), escape(&self.alt_text)
             )),
-            MediaCategory::Audio => SafeString::new(&format!(
+            MediaCategory::Audio => SafeString::trusted(&format!(
                 r#"<div class="media-preview audio"></div><audio src="{}" title="{}" controls></audio>"#,
                 url, escape(&self.alt_text)
             )),
-            MediaCategory::Video => SafeString::new(&format!(
+            MediaCategory::Video => SafeString::trusted(&format!(
```

Comment:
title can't be trusted (and would be unnecessary here if we keep video/audio allowed)